drivers/performance_measurement/app/models/performance_measurement/document_exports/report_export.rb

```diff
@@ -25,10 +25,20 @@ def authorized?
 
     def perform
       with_status_progression do
-        template_file = 'performance_management/warehouse_reports/report/index_pdf'
+        template_file = 'performance_measurement/warehouse_reports/reports/show_pdf'
+        # https://stackoverflow.com/questions/55865582/set-dynamic-header-and-footer-data-on-pdf-generation-from-rails-grover-gem
         PdfGenerator.new.perform(
-          html: view.render(file: template_file, layout: 'layouts/performance_management'),
+          html: view.render(file: template_file, layout: 'layouts/performance_report'),
```

Comment:
Should the layout name match the template name `performance_measurement`?

Reply:
yes!  I've mis-typed that dozens of times.  Thanks

Reply:
Oh, actually, I miss-read the diff.  That's correct as is.  It's re-using the layout from the other performance reports.